Week3/script.js

```diff
@@ -54,192 +52,201 @@ console.log(favoriteAnimals);
-
+  console.log('a ' + color + type);
 }
 
-vehicleType("blue", 2);
-
-        // 5
-
-console.log(3 === 3 ? "yes" : "no")
+vehicleType('blue', 2);
 
-        // 6
+// 5
 
+console.log(3 === 3 ? 'yes' : 'no');
 
+// 6
 
-function vehicle(color,code,age) {
-  let type,condition = "";
+function vehicle(color, code, age) {
+  let type,
+    condition = '';
 
   if (code === 1) {
-    type = " car";
-  } els if (code === 2) {
-    type = " motorbike";
-  } 
+    type = ' car';
+  } else if (code === 2) {
+    type = ' motorbike';
+  }
+
   if (age >= 2) {
-    condition = " used";
-  } else if ((age >= 0) && (age <= 1)) {
-    condition = " new";
+    condition = ' used';
+  } else if (age >= 0 && age <= 1) {
+    condition = ' new';
   }
-  
-  console.log("a " + color + condition + type);
 
+  console.log('a ' + color + condition + type);
 }
-vehicle("blue",1,5);
+vehicle('blue', 1, 5);
 
-        // 7
+// 7
 
-let vehicles = [" motorbike", " caravan", " bike", " car"];
+let vehicles = ['motorbike', 'caravan', 'bike', 'car'];
 
-        // 8
+// 8
 
 console.log(vehicles[2]);
 
-        // 9
+// 9
 
-function newVehicle(color,code,age) {
-  let type,condition = "";
+function newVehicle(color, code, age) {
+  let type,
+    condition = '';
   if (code === 1) {
-    type = vehicles[0];
-  } if (code === 2) {
-    type = vehicles[1];
-  }  if (code === 3) {
-    type = vehicles[2];
-  } if (code === 4) {
-    type = vehicles[3];
-  } if (age >=2) {
-    condition = " used";
-  } if ((age >= 0) && (age <=1)) {
-    condition = " new";
+    type = vehicles[code - 1];
+  }
+  if (code === 2) {
+    type = vehicles[code - 1];
+  }
+  if (code === 3) {
+    type = vehicles[code - 1];
+  }
+  if (code === 4) {
+    type = vehicles[code - 1];
+  }
+  if (age >= 2) {
+    condition = ' used';
+  }
+  if (age >= 0 && age <= 1) {
+    condition = ' new';
   }
-  
-  console.log("a " + color + condition + type)
-}
-newVehicle("green",3,1);
-
-        // 10
-let advertisement = "Amazing Joe's Garage, we service "
-function reAdvertisement(advertisement, vehicles){
-
-  for( let i = 0; i < vehicles.length ;i++){
-    const lastElement = vehicles[vehicles.length-1];
-    const beforLastElement = vehicles[vehicles.length-2];
-    const currentElement = vehicles[i];
 
-    if (currentElement === lastElement) {
-      advertisement = advertisement + currentElement + ".";
-    } else if (currentElement === beforLastElement) {
-      advertisement = advertisement + currentElement + " and ";
+  console.log('a ' + color + condition + ' ' + type);
+}
+newVehicle('green', 3, 1);
+
+// 10
+let advertisement = "Amazing Joe's Garage, we service ";
+function reAdvertisement(advertisement, vehicles) {
+  for (let i = 0; i < vehicles.length; i++) {
+    // const lastElement = vehicles[vehicles.length - 1];
+    // const beforLastElement = vehicles[vehicles.length - 2];
+    // const currentElement = vehicles[i];
+
+    if (i === vehicles.length - 1) {
```

Comment:
Ah nice! This looks very good, I think you forgot to remove the `commented` code on line 171, 172 and 173